modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/spaces/NewSpace.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import Form, {ClayInput} from '@clayui/form';
+import ClayLayout from '@clayui/layout';
+import {useFormik} from 'formik';
+import {navigate} from 'frontend-js-web';
+import React from 'react';
+
+import SpaceService from '../../structure_builder/services/SpaceService';
+import {FieldText} from '../components/forms';
+import {required, validate} from '../components/forms/validations';
+import {CreateSpaceStepOneIllustration} from './CreateSpaceStepOneIllustration';
+import {NewSpaceFormSection} from './NewSpaceFormSection';
+
+export interface NewSpaceProps {
+	baseRedirectUrl: string;
+}
+
+const NewSpace = ({baseRedirectUrl}: NewSpaceProps) => {
+	const {errors, handleChange, handleSubmit, isSubmitting, touched, values} =
+		useFormik({
+			initialValues: {
+				description: '',
+				name: '',
+			},
+			onSubmit: (values) => {
+				const {description, name} = values;
+
+				SpaceService.addSpace({description, name}).then((response) => {
+					navigate(baseRedirectUrl + response.id);
+				});
+			},
+			validate: (values) =>
+				validate(
+					{
+						name: [required],
+					},
+					values
+				),
+		});
+
+	return (
+		<ClayLayout.Row className="p-4">
+			<NewSpaceFormSection
+				description={Liferay.Language.get(
+					'spaces-are-essential-for-organizing-defining-and-managing-your-content-and-files'
+				)}
+				linkLabel={Liferay.Language.get('learn-more-about-spaces')}
+
+				// TODO: update this link
+
+				linkUrl="/"
```

Comment:
I need the URL for this link:
<img width="706" alt="Screenshot 2025-05-15 at 18 36 00" src="https://github.com/user-attachments/assets/fa00767a-1ba4-4f0c-97c3-76b5ed36fa33" />